web/src/pages/apd/proposed-budget/IncentivePayments.test.js

```diff
@@ -1,5 +1,7 @@
-import { shallow } from 'enzyme';
 import React from 'react';
+import {
+  renderWithConnection
+} from 'apd-testing-library';
 
 import {
   plain as IncentivePayments,
```

Comment:
if you use the plain version instead of 
```
import IncentivePayments from '../../../redux/actions/editApd'
```
then you aren't really testing the connected component. You could rewrite these tests to use the connected component and then rewrite your set up like
```
const setup = (props = {}) => {
  renderWithConnection(<IncentivePayments {...defaultProps} {...props} />, {
    initialState: {
      apd: {
        data: {
          proposedBudget: {
            incentivePayments: {
              ehAmt: { 2022: [1, 2, 3, 4], 2023: [5, 6, 7, 8] },
              ehCt: { 2022: [9, 10, 11, 12], 2023: [13, 14, 15, 16] },
              epAmt: { 2022: [17, 18, 19, 20], 2023: [21, 22, 23, 24] },
              epCt: { 2022: [25, 26, 27, 28], 2023: [29, 30, 31, 32] }
            }
          }
        }
      }
    }
  });
};
```
you won't need to calculate the totals because the systems *should* do that for you

Reply:
Absolutely!